Source/NSUnit.m

```diff
@@ -0,0 +1,1770 @@
+                                                  coefficient: 764.555
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) cubicMiles 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"mi^3"
+                                                  coefficient: 4.168e+12
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) acreFeet 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"af"
+                                                  coefficient: 1.233e+6
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) bushels 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"bsh"
+                                                  coefficient: 32.2391
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) teaspoons 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"tsp"
+                                                  coefficient: 0.00492892
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) tablespoons 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"tbsp"
+                                                  coefficient: 0.0147868
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) fluidOunces 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"fl oz"
+                                                  coefficient: 0.295735
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) cups 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"cups"
+                                                  coefficient: 0.24
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) pints 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"pt"
+                                                  coefficient: 0.473176
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) quarts 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"qt"
+                                                  coefficient: 0.946353
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) gallons 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"gal"
+                                                  coefficient: 3.78541
+                                                     constant: 0.0];
+  return result;
+}
+
++ (NSUnitVolume *) imperialTeaspoons 
+{
+  NSUnitVolume *result = [[NSUnitVolume alloc] initWithSymbol: @"tsp"
```

Comment:
Here the same symbol as for normal teaspoons gets reused. Apple documented it that way, but for me this makes no sense.

Reply:
Also, called for in the docs.   We could agree on a different symbol.

Reply:
Unfortunately, this is per documentation.